compiler/rustc_target/src/spec/mod.rs

```diff
@@ -521,7 +521,7 @@ supported_targets! {
     ("thumbv7neon-linux-androideabi", thumbv7neon_linux_androideabi),
```

Comment:
On commit message: typo in "hyperverisers" and "dependenices", duplicated "situations", the -> that, uses -> use.

The GitHub interface also shows me a second, huge/spurious commit, for whatever reason.